swagger.yaml

```diff
@@ -18,17 +18,28 @@ consumes:
 produces:
   - text/plain
 paths:
-  /:
+  /getSpendingAuthorit:
     get:
       summary: Get information
       description: Build and runtime information about the application.
       parameters:
-        - $ref: '#/parameters/pretty'
+        - in: "query"
+          name: onid
+          required: true
+          type: string
       responses:
         200:
-          description: Object containing information
+          description: "request recived"
```

Comment:
Can you give a more descriptive message here? "request received" also sounds not right for the `200` response.